pages/dao/[symbol]/proposal/components/instructions/WithdrawInsuranceFromMangoDepository.tsx

```diff
@@ -68,12 +69,13 @@ const WithdrawInsuranceFromMangoDepository = ({
       wallet?.publicKey
     ) {
       const createIx = await createWithdrawInsuranceFromMangoDepositoryInstruction(
-        connection.current,
+        connection,
         form.governedAccount?.governance.info.governedAccount,
         form.governedAccount?.governance.pubkey,
         new PublicKey(form.collateralMint),
         new PublicKey(form.insuranceMint),
-        form.insuranceWithdrawnAmount || 0
+        form.insuranceWithdrawnAmount || 0,
```

Comment:
If we can avoid default values and just error, not sure idk typescript

Reply:
resolved